docs/INSTALL.md

```diff
@@ -0,0 +1,214 @@
+   - **Product Name**: Provide your product name
+   - **Module**: UpCloud VPS
+   - **Create as Hidden**: Off
+3. Click on Save Changes
+
+### Module Settings
+
+On the next page:
+1. Verify the Product Type
+2. Set Welcome Email to "Dedicated/VPS Server Welcome Email"
+3. Go to the Module Settings Tab
+4. Select the required details:
+   - Module Name
+   - Server group
+   - Default Location
+   - Plan
+   - Template
+5. If for Plans you selected "Custom", then users will have the ability to choose their own CPU, RAM, and Storage space based on which VMs will be created.
+
+![Server Plan Config](images/server_plan_config.png)
+
+6. Decide whether the product requires manual approval from admins or is automatically deployed upon order.
+7. Click the Save Changes button.
+
+![Server Plan Ready](images/server_plan_ready.png)
+
+This will create custom fields named (`instanceId`, `ssh_key` & `userData`) as shown below:
+
+![Custom Fields](images/custom_fields.png)
+
+### Configurable Options
+
+The module will automatically create Configurable Options when you create your first product. You can adjust these options as needed.
+
+#### If No Custom Plan is Selected
+
+1. Go to System Settings -> Configurable Options and click on edit
+2. Edit the newly created Options
+
+![Configurable Options](images/configurable_options.png)
+
+
+3. For Configurable Option "Location - template|Template", adjust the prices for Windows OS
+
+![OS prices](images/os_prices.png)
+
+4. For Configurable Option "location|Location", adjust as needed
+
+![Location](images/location.png)
+
+
+#### If Custom Plan is Selected
+
+If custom plan is selected, then a separate Configurable Option group is created with name "Configurable options for UpCloud - Custom Plans" and you will have additional options:
+- **Memory**: Users can choose the memory of a VPS, ranging from 4GB to 64GB.
+- **vCPU**: Users can choose the vCPU of a VPS, ranging from 2 cores to 24 cores.
+- **Storage**: Users can choose the storage of a VPS, ranging from 50GB to 460GB.
+
+![Configurable Options Custom](images/configurable_options_custom.png)
+
+## Adding Existing VPS to Users
+
+To add an existing UpCloud VPS to a user's account:
+1. Edit the user's product
+2. Within the custom field named "Instance Id", add your VPS UUID that you can obtain from UpCloud Hub
+
+![Existing VPS](images/existing_vps.png)
+
+## Additional Configuration
+
+### Product/Service -> Custom Fields
+
+- **User Data/Cloud-init**: If provided by the user, these scripts will initiate after VM creation to kickstart the initialization process.
+- **SSH Key**: If supplied by the user during VM creation, it will be utilized for user authentication instead of a password; otherwise, the system will default to using a password for authentication. Note that some templates only support SSH keys.
+- **Instance Id**: This field is used to assign a user with a VPS
+
+### System Settings -> Configurable Options
```

Comment:
An update to this wrt recent changes re #68 would be in order here, too.

Reply:
Thanks, I moved this to the section above about product config & configurable options